File: Company/src/main/webapp/service/nalogZaPrenosService.js
/**
 * Created by Olivera on 22.6.2017..
 */

app.service('nalogZaPrenosService', function($http){
    return {
        read: function(onSuccess, onError){
            $http.get('/api/naloziZaPrenos').then(onSuccess, onError);
        },
        kreirajNalog: function (faktura, onSuccess, onError) {
            alert("Bla")
            $http.post('api/naloziZaPrenos/kreirajNalog', faktura).then(onSuccess, onError);
        }
    }
});

File: Company/src/main/java/com/ftn/service/NalogZaPrenosService.java
package com.ftn.service;

import com.ftn.model.dto.NalogZaPrenosDTO;

import java.util.List;

/**
 * Created by Olivera on 22.6.2017..
 */
public interface NalogZaPrenosService {

    List<NalogZaPrenosDTO> read();

    NalogZaPrenosDTO create(NalogZaPrenosDTO nalogZaPrenosDTO);
}
